resources/dicts/events/death/general.json

```diff
@@ -4095,7 +4303,7 @@
         ],
         "tags": [],
         "weight": 20,
-        "event_text": "m_c failed to interpret a warning sign from StarClan and lost a life as a result.",
+        "event_text": "m_c failed to interpret a warning sign from StarClan.",
```

Comment:
I think we should add a "and couldn't foresee the circumstances of their death." to better explain why they've suddenly lost a life, otherwise i think we'd end up with some confused players haha